global.d.ts

```diff
@@ -1 +1,20 @@
 /// <reference types="@docusaurus/theme-classic" />
+
+declare module '@theme/ProgressiveImage' {
```

Comment:
i don't think we need this declaration here, please revert this part. I understand that typescript doesn't compile sucessfuly otherwise, but we mostly care about the docusaurus build process working - not type checking. I don't want to maintain this, so let's remove it. the other file looks great.